Automattic/jetpack
# CLI: Changelog Passthrough

Adds ability to passthrough arguments directly to changelogger from Jetpack CLI. 

#### Changes proposed in this Pull Request:

* Adds ability to passthrough arguments directly to changelogger from Jetpack CLI.

#### Jetpack product discussion


p9dueE-2A1-p2
#### Does this pull request change what data or activity we track or use?


N/a
#### Testing instructions:






* Try out the changelogger through jetpack's CLI by passing arguments to it:
	* -f "file" (name of changelog file - defaults to branch name and can be omitted)  
	* -s "significance"
	* -t "type"
	* -e "entry" (the changelog comment itself)
* ex: `jetpack changelog add packages/abtest -s "patch" -t "security" -e "My great message"`
* If not all arguments are passed, it should default to interactive mode (unless you're also passing a file name in which case it will just give you a specific error about what you're missing - maybe we want this to be default anyway).

No need to specify `--no-interactive` - it does that when all arguments are passed. We can talk about if we want this to be the case, but if you're already passing all the arguments imo you're gonna want non interactive mode.
